tests/test_zonal.py

```diff
@@ -4,13 +4,14 @@
 import sys
 
 import numpy as np
+import geopandas as gpd
```

Comment:
I see in the test you runned that there are problems with geopandas import. I don't know what to do to solve this. Maybe I was not supposed to add this line to the test file ?